From 12882c775c84ceb8317d1492f0391c1d24c0bb73 Mon Sep 17 00:00:00 2001
From: Markus Rothe <markus.rothe@rite.cc>
Date: Wed, 11 Mar 2020 08:12:36 +0000
Subject: [PATCH] CMakeLists.txt: use proper libdir

---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 58a72c8..c2597c2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -118,13 +118,13 @@ configure_file("${CMAKE_SOURCE_DIR}/cmake/${PROJECT_NAME}-config.cmake"
 export(EXPORT "${PROJECT_NAME}-targets" NAMESPACE AWS::)
 
 install(EXPORT "${PROJECT_NAME}-targets"
-    DESTINATION "lib/${PROJECT_NAME}/cmake/"
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/${PROJECT_NAME}/cmake/"
     NAMESPACE AWS::)
 
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config.cmake"
     "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config-version.cmake"
-    DESTINATION "lib/${PROJECT_NAME}/cmake/")
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/${PROJECT_NAME}/cmake/")
 
 install(PROGRAMS "${CMAKE_SOURCE_DIR}/packaging/packager"
-    DESTINATION "lib/${PROJECT_NAME}/cmake/")
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/${PROJECT_NAME}/cmake/")
 
-- 
2.24.1

